### ISSUE TITLE
Searching Projects with require publications = yes

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#2615]
In testing ArctosPG I searched for projects that have publications.

![image](https://user-images.githubusercontent.com/5725767/80318698-16dd4e80-87c9-11ea-8851-a3bee13a5a80.png)

The results included 251 projects and no publications. 

![image](https://user-images.githubusercontent.com/5725767/80318774-aa168400-87c9-11ea-8366-ac8994739458.png)

While logically, this makes sense - publications are not projects, therefor by definition they aren't what I am searching, but from a discovery standpoint, it is disappointing that all of the publications associated with the projects in the results aren't there for me to easily see.

Not sure if anyone else thinks it would be useful to see those publications, but thought I would put this here for discussion.

**Comment by dustymc** [comment]
I think that's the form itself trying to do too much - I've semi-merged projects and publications in an effort to better get people to usage data, that requires a bit of "black box," that gets a bit weird in some situations - I think mostly when looking for specifics rather than an overview.

Or maybe we just need project publication summary data displayed on the results form?

**Comment by campmlc** [comment]
I agree that we should see the publications if possible. Otherwise what's
the point of the search. Would a summary page dashboard view be possible?

**Comment by dustymc** [comment]
>Otherwise what's the point of the search.

The search did precisely what was asked of it. Pushing project-related publications to the publications pane would be an addition. (Possibly a very weird one if you're just looking for publications; then you'd likely want to exclude the ones that are proposed to get magicked in from publications.)

>Would a summary page dashboard view be possible?

Not sure what that means, but given a mock-up or functional requirements or something of the sort, probably.

**Comment by Jegelewicz** [comment]
> The search did precisely what was asked of it.

Agree completely, but honestly, why would I search for projects that include publications, but then not want to know what those publications are?

**Comment by dustymc** [comment]
If you want publications, then search by something that publications can have!

(I get it, I think, sorta, but see above - this looks like a nightmare when you're looking for publications that carry shared data and not those that are just attached to projects that carry whatever you searched for. Maybe that's some sort of "magic_dig_deeper()" checkbox or something??)
